### ISSUE TITLE
I dont see where to launch the window.

### ISSUE THREAD

**Issue by cdavies2014** [ttaubert/customizable-shortcuts#104]
It's not in the add-ons. It's not in the customize menu. or Preferences.
Don't answer this. Update help.

**Comment by ttaubert** [comment]
Please install verseion 0.9.6 from here:

https://github.com/ttaubert/customizable-shortcuts/blob/master/customizable-shortcuts@timtaubert.de-0.9.6.xpi?raw=true

**Comment by gene1wood** [comment]
@ttaubert What do you think about leaving one of these github issues in an `open` state until the new version is uploaded to firefox add-ons? I ask because I experienced the same problem as others here are mentioning, went to github to look to see if anyone was experiencing this issue and saw nothing. It was only after reading through comments on add-ons did I see people mentioning that the issue is that github has the fix but it hasn't been published to add-ons.

It might help other people if there was an open github issue that they would see (at least until the fix is uploaded to add-ons and this is no longer a problem).

**Comment by gene1wood** [comment]
Or perhaps open a new issue that just calls out the version of Firefox and version of Customizable Shortcuts that don't work together, and the fact that you can use the [0.9.7 release](https://github.com/ttaubert/customizable-shortcuts/blob/master/customizable-shortcuts@timtaubert.de-0.9.7.xpi?raw=true) to workaround it

**Comment by ttaubert** [comment]
Happy to let you know the add-on has finally been fully reviewed.

**Comment by gene1wood** [comment]
Sweet!
